File: WebService/Entities/Products/FetchProductResponse.php
<?php
class FetchProductResponse
{
    public function __construct(
        public readonly int $productId,
        public readonly string $productName,
        public readonly string $productDescription,
        public readonly float $price,
        public readonly int $productCategory,
        public readonly string $productImage, // base64
    ) {
    }
}

?>

File: app/src/main/java/com/example/mobilepose/Model/API/Entities/FetchProductResponse.java
package com.example.mobilepose.Model.API.Entities;

import com.google.gson.annotations.SerializedName;

public class FetchProductResponse {
    @SerializedName("productId")
    public int productId;
    @SerializedName("productName")
    public String productName;
    @SerializedName("productDescription")
    public String productDescription;
    @SerializedName("price")
    public float price;
    @SerializedName("productCategory")
    public int productCategory;
    @SerializedName("productImage")
    public String productImage; // base64
    @SerializedName("available")
    public int available;


}
